sqldelight-compiler/src/main/kotlin/com/squareup/sqldelight/core/compiler/SqlDelightCompiler.kt

```diff
@@ -103,6 +103,17 @@ object SqlDelightCompiler {
         }
   }
 
+  private val NamedElement.nomalizedName: String
+    get() {
+      val f = name[0]
+      val l = name[name.lastIndex]
+      return if (f in "\"'`" && f == l || f == '[' && l == ']') {
```

Comment:
can you wrap the two sides of the or statement in parens

Reply:
Done